content/en/database_monitoring/setup_mongodb/amazon_documentdb.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Setting Up Database Monitoring for Amazon DocumentDB
+further_reading:
+    - link: '/integrations/mongo/'
+      tag: 'Documentation'
+      text: 'Basic MongoDB Integration'
+---
+
+Database Monitoring offers comprehensive insights into your DocumentDB databases by providing access to critical metrics, operation samples, explain plans, and replication state changes. To take advantage of Database Monitoring for Amazon DocumentDB, ensure that the Datadog Agent is installed and configured to connect to your DocumentDB instances. This guide outlines the steps to set up Database Monitoring for Amazon DocumentDB.
+
+## Before you begin
+
+Supported Amazon DocumentDB major versions
+: 4.4, 5.0
+
+{{% dbm-mongodb-before-you-begin %}}
+
+## Setup
+
+To enable Database Monitoring for your database:
+
+1. [Grant the Agent access to your MongoDB instances](#grant-the-agent-access-to-your-mongodb-instances)
+2. [Install and configure the Agent](#install-and-configure-the-agent)
+3. [(Optional) Install the Amazon DocumentDB integration](#install-the-amazon-documentdb-integration)
+
+### Grant the Agent access to your MongoDB instances
+
+The Datadog Agent requires read-only access to the Amazon DocumentDB instance to collect statistics and queries.
+
+{{< tabs >}}
+{{% tab "Standalone" %}}
```

Comment:
no need to have standalone for docdb

Reply:
I saw in aws that you can create a docdb cluster with one instance

Reply:
yeah even with one instance it's a replica in docdb